app/models/grader_permission.rb

```diff
@@ -0,0 +1,5 @@
+# Contains the grader permissions for a particular grader
+class GraderPermission < ApplicationRecord
+  self.table_name = 'grader_permission'
```

Comment:
The table name should be 'grader_permissions'. Please stick with the default naming conventions. You will have to change this in your migration files as well.